olmo/config.py

```diff
@@ -987,6 +987,11 @@ class TrainConfig(BaseConfig):
     How often to log to the console.
     """
 
+    gen1_gc_interval: int = 1
```

Comment:
nit: it might be nice to provide an easy way to turn this off (e.g. setting it to None)

Reply:
Done! 989f7992

(I assumed you mean disabling manual collection, i.e. not messing with it at all)